Framework/Algorithms/src/SumSpectra.cpp

```diff
@@ -379,7 +413,7 @@ void SumSpectra::doRebinnedOutput(MatrixWorkspace_sptr outputWorkspace,
   // Loop over spectra
   for (const auto i : m_indices) {
     // Don't go outside the range.
-    if ((i >= m_numberOfSpectra) || (i < 0)) {
+    if ((i >= this->m_numberOfSpectra) || (i < 0)) {
```

Comment:
The `this->` thing is used inconsistently throughout this file anyhow. Personally, I'd remove them as my IDE color-codes member variables automatically but your mileage my vary...